draft-dekater-scion-controlplane.md

```diff
@@ -1356,9 +1361,9 @@ All of these are manageable on a present day small server or desktop machine.
 For much larger, more highly connected ASes, the path-discovery tasks of the control service can be distributed over many instances in order to increase the PCB throughput.
 
 
-On a cold start of the network, full connectivity is obtained after a number of propagation steps corresponding to the diameter of the network. Assuming a network diameter of 6, this corresponds to roughly 3 minutes on average.
+On a cold start of the network, full connectivity is obtained at worst after a number of propagation steps corresponding to the diameter of the network. Assuming a network diameter of 6, this corresponds to roughly 3 minutes on average.
```

Comment:
Remove this "at worst"; can't be fewer steps (even if they'd be triggered quicker)

Reply:
right, I'll do